### DISCUSSION TITLE
AI的運用在修圖上

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS4719-C/Python-Community-GS4719-C#753] [科技新知]
這次軟體的更新，便是運用了so的技術結合現有的功能去加強，由此可見這項技術的運用將越來越廣泛，也讓生活更加豐富與便利。

https://digiphoto.techbang.com/posts/11799-ai-blessing-after-the-system-is-easier-adobe-lightroom-classic-version-114-is-a-retouching-demonstration-of-major-new-features

**Comment by williamyen0623** [comment]
與真實不符的照片又要增加了

**Comment by ghost** [comment]
感覺可以做成app了，手機拍完直接修圖

**Comment by evanchou1204** [comment]
直接做出日出跟夕陽也太猛，根本看不出來是AI做的...

**Comment by danwan1027** [comment]
AI 的繪畫能力越來愈強了...

**Comment by Pietra1226** [comment]
以後AI魚目混珠說是攝影師拍的的我也信了，攝影師會不會消失了⋯
